package.json

```diff
@@ -23,10 +23,18 @@
     "@types/react": "16.8.8",
     "@types/react-dom": "16.8.2",
     "babel-loader": "^8.0.5",
+    "css-loader": "^2.1.1",
+    "eslint": "^5.3.0",
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
You've added eslint! It's cool! But you also should configure it) I assure you that It'll help you to avoid a lot of comments from my side.